app/controllers/api/v1/admin/reports_controller.rb

```diff
@@ -14,16 +14,13 @@ class Api::V1::Admin::ReportsController < Api::BaseController
   after_action :verify_authorized
   after_action :insert_pagination_headers, only: :index
 
-  FILTER_PARAMS = %i(
-    resolved
-    account_id
-    target_account_id
-  ).freeze
-
-  PAGINATION_PARAMS = (%i(limit) + FILTER_PARAMS).freeze
+  PAGINATION_PARAMS = (%i(limit) + ReportFilter::KEYS).freeze
 
   def index
     authorize :report, :index?
+
+    return redirect_to api_v1_admin_reports_path filter_params.merge(status: 'resolved') if params[:resolved].present?
```

Comment:
I think a redirect here is what makes most sense, since many http clients will automatically follow redirects, allowing them to continue working.

Reply:
Since we didn't originally support search here, we don't need to support quite so many redirects.